frontend/src/pages/Admin/index.js

```diff
@@ -0,0 +1,214 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import _ from 'lodash';
+import {
+  GridContainer, Label, TextInput, Grid, SideNav, Button,
+} from '@trussworks/react-uswds';
+import UserSection from './UserSection';
+import NavLink from '../../components/NavLink';
+
+// Fake return from an API
+const fetchedUsers = [
+  {
+    id: 1,
+    email: 'dumbledore@hogwarts.com',
+    jobTitle: undefined,
+    fullName: undefined,
+    permissions: undefined,
+    region: undefined,
+  },
+  {
+    id: 2,
+    email: 'hermionegranger@hogwarts.com',
+    jobTitle: 'Systems Specialist',
+    fullName: 'Hermione Granger',
+    permissions: [
+      {
+        region: 0,
+        scope: 'SITE_ACCESS',
+      },
+      {
+        region: 1,
+        scope: 'READ_WRITE_REPORTS',
+      },
+    ],
+    region: '1',
+  },
+  {
+    id: 3,
+    email: 'harrypotter@hogwarts.com',
+    jobTitle: 'Grantee Specialist',
+    fullName: 'Harry Potter',
+    permissions: [
+      {
+        region: 0,
+        scope: 'ADMIN',
+      },
+      {
+        region: 0,
+        scope: 'SITE_ACCESS',
+      },
+      {
+        region: 1,
+        scope: 'READ_REPORTS',
+      },
+      {
+        region: 1,
+        scope: 'READ_WRITE_REPORTS',
+      },
+      {
+        region: 2,
+        scope: 'READ_REPORTS',
+      },
+      {
+        region: 3,
+        scope: 'READ_REPORTS',
+      },
+      {
+        region: 4,
+        scope: 'READ_REPORTS',
+      },
+      {
+        region: 5,
+        scope: 'READ_REPORTS',
+      },
+      {
+        region: 6,
+        scope: 'READ_REPORTS',
+      },
+      {
+        region: 7,
+        scope: 'READ_REPORTS',
+      },
+      {
+        region: 8,
+        scope: 'READ_REPORTS',
+      },
+      {
+        region: 9,
+        scope: 'READ_REPORTS',
+      },
+    ],
+    region: '1',
+  },
+  {
+    id: 4,
+    email: 'ronweasley@hogwarts.com',
+    jobTitle: 'Program Specialist',
+    fullName: 'Ron Weasley',
+    permissions: [
+      {
+        region: 0,
```

Comment:
Is there literally going to be a region called `0` or is this a `null` value?

Reply:
Good question, I got so used to region 0 that it stopped bothering me until you pointed it out. I haven't given much thought on if the region is 0 or null, _seems_ like something that will get nailed down in https://github.com/HHS/Head-Start-TTADP/issues/50. Having region be 0 or null doesn't feel that great to me, maybe in https://github.com/HHS/Head-Start-TTADP/issues/50 we decide to have a table for regional permissions (with a foreign key to region) and a table for global permissions (with no foreign key to region). Maybe we only have a single global permission so it makes sense to add a column directly to the users table. Regardless I'll add a comment to all the places where `region 0` is being used explaining it isn't a final decision.